### DISCUSSION TITLE
S403 compatible connector

### DISCUSSION THREAD

**Discussion by Sytten** [revk/xESP32-Faikout#281] [Q&A] [answered]
I am trying to figure out which connector (JST or other) would fit on the S403. If anyone has suggestions :)

EDIT: I ended up buying an adaptor board here https://www.ebay.com/itm/225477498927 for those interested, will update when it comes up.

![PXL_20240205_023814386.jpg](https://github.com/revk/ESP32-Faikin/assets/2366731/ae269fb0-36d3-412f-afc1-13ed9ef3b0f1)

**Comment by revk** [comment]
Be careful as I believe S403 is likely to be "live" when air-con is powered on. This would make the whole Faikin module live.

**Reply by Sytten** [reply]
It is live I am aware so keeping it inside the metal housing for sure.

**Reply by Sytten** [reply]
I decided to buy the adaptor board but this thread has been super helpful. Now I just need to buy an JST EH cable.

**Reply by 44616E** [reply]
How much was the board? I have two units running off this port.
I could make you a cable, if you can't find one.

**Reply by Sytten** [reply]
I found the board on ebay for 68USD each, I figured this was worth for the optocouplers to not fry my machine (the whole thing costs like 4.5k each machine). I would not have paid the quoted price from the installer since it was like 250USD each. 
I was thinking on going on aliexpress to buy one since it is a simple JST EH on each side, but the guy I bought it from seems to have some in stock.
https://www.ebay.com/itm/225477498927

**Reply by 44616E** [reply]
Well so far I have them in 5 AC and not had an issue. Have 8 more to install.

**Reply by Sytten** [reply]
Yep I am 100% convinced that it is not necessary but I am not currently equiped to make custom connectors and cables of that size so now I think I am all set. The guy even gave me the official daikin cables for the S21 port.

**Comment by njh** [comment]
Are you able to measure the pitch? (distance between each pin)
I measured mine by gently pressing some "Blu Tack" on the connector and then measuring that.

**Reply by Sytten** [reply]
Mine is in a very bad location to do this unfortunately

**Reply by 44616E** [reply]
Mine was 2.5mm used venier calipers.

**Comment by 44616E** [comment, marked as answer]
Connector is XARP-10V
Contact: SXA-001T-P0.6

At least this for my unit. But be very careful this port is mains potential. Power off and wait 30 mins before connecting / opening and test for dead / no voltage and make sure the faikin board is insulated to 500V and can not be touch plus additional precautions. This can be lethal.

**Reply by Sytten** [reply]
Thanks, I am aware of the risks. Keeping the module inside the metal box should be fine. But  I might buy the insulating module, I just wished it wasnt so damn expensive.

**Reply by Sytten** [reply]
Is your unit similar to an FTX12NMVJU? I know this connector comes in 2.5mm and 2.54mm pitch

**Reply by 44616E** [reply]
I heat shrinked the module and used 300V double insulated cable

**Reply by revk** [reply]
Good

**Reply by 44616E** [reply]
Mine is a FTXF25W for reference.
